src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -558,6 +556,21 @@ public void saveSubscriptions(String subscriptionsJson, boolean reconcileCapacit
     }
   }
 
+  @Transactional
```

Comment:
I'm seeing a warning in the IDE that `methods annotated with '@Transactional' must be overridable`.  The `@Transactional` won't do anything here either since the call is made internally.  See https://www.baeldung.com/transaction-configuration-with-jpa-and-spring#pitfalls and the discussion on proxy objects.